base_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class BaseDecorator < Nameable
+  def initialize(nameable)
+    super
+    @nameable = nameable
+  end
```

Comment:
- [ ] Well done for adding the constructor 🥇 However, when I am testing this, the following error is occurring⚠👇 Kindly fix this ✨
 
  ![image](https://github.com/danielamoreno699/school-library/assets/106119105/bbcafa4f-caf8-49d9-8942-bb5cefc9c672)